packages/mobile/src/identity/privateHashing.ts

```diff
@@ -247,10 +249,19 @@ function* navigateToQuotaPurchaseScreen() {
       throw new Error('Purchase tx failed')
     }
 
+    ValoraAnalytics.track(AnalyticsEvents.phone_number_quota_purchase_success)
     Logger.debug(`${TAG}@navigateToQuotaPurchaseScreen`, `Quota purchase successful`)
     navigateBack()
     return true
   } catch (error) {
+    // CC Rossy
```

Comment:
@jmrossy does this look right to you? i don't think we were distinguishing between a skip and fail in the previous iteration

Reply:
Looks okay. It might be more natural to just `reject('skipped')` and then compare the error to 'skipped' directly but this is fine if you've already tested this.